libchisel/src/utils.rs

```diff
@@ -0,0 +1,33 @@
+//! These are helpers to be used internally.
+
+use parity_wasm::elements::{deserialize_buffer, serialize, Module};
+
+pub trait HasNamesSection {
+    /// Returns true if the module has a NamesSection.
+    fn has_names_section(&self) -> bool;
+}
+
+impl HasNamesSection for Module {
+    fn has_names_section(&self) -> bool {
+        // Lets forcefully parse in case it wasn't yet.
+        let mut module = self.clone().parse_names().unwrap();
```

Comment:
I would prefer an `expect()` here rather than unwrap.

Reply:
Yeah fair, forgot to change it amidst copy pastes.